mirest/stylist-api
# #165026611 Create user model

#### What does this PR do?
 - It creates the user  `model`

#### Description of Task to be completed?
- Create user model
- Write tests for the model
#### How should this be manually tested?
- Pull the branch `git fetch origin ch-create-user-modal-165026611` 
- Check it out `git checkout ch-create-user-modal-165026611`
- Run `coverage run --source='apps' manage.py test`

#### Pivotal Tracker Story
- [#165026611](https://www.pivotaltracker.com/story/show/165026611)
